client/src/components/Column.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
+import TaskCard from "./TaskCard";
+//check how to use Cards in column data
+
+//materialUi
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Card from "@material-ui/core/Card";
+import Grid from "@material-ui/core/Grid";
+import CardContent from "@material-ui/core/CardContent";
+import CancelIcon from "@material-ui/icons/Cancel";
+
+//Drag and Drop
+import { Droppable, Draggable } from "react-beautiful-dnd";
+
+const Column = props => {
+  const classes = useStyles(props);
+  const { column, tasks, createNew, index } = props;
+
+  const [testSt, setTextSt] = useState({
+    newTaskName: "",
+    newColumnTitle: ""
+  });
+  const { newTaskName, newColumnTitle } = testSt;
+
+  console.log(testSt);
+
+  const onChange = e => {
+    setTextSt({
+      ...testSt,
+      [e.target.name]: e.target.value
+    });
+  };
+
+  const createNewColumn = e => {
+    e.preventDefault();
+    //Send newColumnTitle to create new column
+  };
+
+  const deleteColumn = e => {
+    e.preventDefault();
+    //Send newColumnTitle to create new column
+  };
+
+  if (createNew) {
+    return (
+      <Card className={classes.addColumn}>
+        <CardContent>
+          <TextField name='newColumnTitle' value={newColumnTitle} onChange={e => onChange(e)}>
+            Enter column Name
+          </TextField>
+          <Button
+            className={classes.btn}
+            type='submit'
+            variant='contained'
+            color='primary'
+            onChange={e => createNewColumn(e)}>
+            Create New column
```

Comment:
you would only use the e for textfield. also you should just remove this whole button and column since were adding columns in a different way

Reply:
Hi, this part removed and added add column feature to the  branch 'FEBE-dashBoard'